tests/cargo-kani/ws-specified/expected

```diff
@@ -0,0 +1 @@
+ ** 1 of 1 failed
```

Comment:
With this expected file, it seems to me that this test would pass even if Kani included the harness in the `bincrate` crate as well? If I understood correctly, the purpose of this test is to check that the `--package` option is processed correctly, and that Kani only runs the specified package. If my understanding is correct, can we check that only one harness ran, by checking the multi-harness summary line that Kani emits, e.g.
```
Complete - 2 successfully verified harnesses, 0 failures, 2 total.
```

Reply:
96a256db64ff6bfdc7ae72bd1adc0af773ed86a7